### PR TITLE
Feature/AUT-17/Support multiple column layout in passage

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-17
Related to: https://github.com/oat-sa/extension-tao-itemqti/pull/1646

How to test:
- go to Passage Authoring
- Check that blocks can be manipulated in the same way as they can in item authoring: dropped above, left, right, below an existing block, resized in 1/12 steps, removed or via the “plus button”.

image

### CODE REVIEW

File: views/js/qtiCreator/widgets/static/text/Widget.js [hunk 1]
```diff
@@ -28,18 +28,18 @@ define([
...
-        //         state : 'active'
-        //     }));
-
-        // this.$container.append($tlb);
-
-        // $tlb.find('[data-role="delete"]').on('click.widget-box', function(e){
-        //     e.stopPropagation();//to prevent direct deleting;
-        //     self.changeState('deleting');
-        // });
+        var self = this,
```
Comment (AnastasiaArcadia) [inline views/js/qtiCreator/widgets/static/text/Widget.js:31]: Do we need to move to ES6?

**Review by AnastasiaArcadia — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not master

File: views/js/qtiCreator/editor/blockAdder/tpl/addColumnRow.tpl [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class="add-block-element">
+    <div class="line"></div>
+    <div class="circle" title="Insert block below"><span>+</span></div>
+</div>
```
Comment (habemuscode) [inline views/js/qtiCreator/editor/blockAdder/tpl/addColumnRow.tpl:4]: End line

File: views/js/qtiCreator/editor/blockAdder/tpl/placeholder.tpl [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<div class="colrow">
+</div>
```
Comment (habemuscode) [inline views/js/qtiCreator/editor/blockAdder/tpl/placeholder.tpl:2]: End line

**Review by habemuscode — APPROVED** [review]
Good job!

- [x]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**